client/src/main/java/org/eclipse/hono/client/HonoClient.java

```diff
@@ -289,14 +291,16 @@ HonoClient createCredentialsClient(
     /**
      * Closes this client's connection to the Hono server.
      * <p>
-     * This method waits for at most 5 seconds for the connection to be closed properly.
+     * This method waits for at most 5 seconds for the connection to be closed properly. Any succeeding attempts to 
```

Comment:
should better be `Any subsequent attempts ...`

Reply:
This was what I searched for :)